ui/src/components/table/CustomTableRow.jsx

```diff
@@ -73,6 +75,16 @@ function CustomTableRow(props) {
                             />
                         </Tooltip>
                     )}
+                    {rowActions.includes('search') && (
+                        <Tooltip content={_('Search')}>
+                            <ActionButtonComponent
+                                appearance="flat"
+                                icon={<Magnifier screenReaderText={null} size={1} />}
+                                onClick={() => handleSearchActionClick(selectedRow)}
```

Comment:
is it not possible to use <a> instead of onClick handler? using onClick is a bad practice because it disables browser's navigation functionality (right clickľ, it is not accessible 

https://splunkui.splunk.com/Packages/react-ui/Button?section=examples#With%20to%20indicator